schemas/com.snowplowanalytics.snowplow/atomic/jsonschema/1-0-0

```diff
@@ -0,0 +1,513 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for an atomic canonical Snowplow event",
+	"self": {
+		"vendor": "com.snowplowanalytics.snowplow",
+		"name": "atomic",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"type": "object",
+	"properties": {
+		"app_id": {
+			"type": ["string", "null"],
+			"maxLength": 255
+		},
+		"platform": {
+			"type": ["string", "null"],
+			"maxLength": 255
+		},
+		"etl_tstamp": {
+			"type": ["string", "null"],
+			"format": "date-time"
+		},
+		"collector_tstamp": {
+			"type": "string",
+			"format": "date-time"
+		},
+		"dvce_created_tstamp": {
+			"type": ["string", "null"],
+			"format": "date-time"
+		},
+		"event": {
+			"type": ["string", "null"],
+			"maxLength": 128
+		},
+		"event_id": {
+			"type": "string",
+			"maxLength": 36
+		},
+		"txn_id": {
+			"type": ["integer", "null"]
+		},
+		"name_tracker": {
+			"type": ["string", "null"],
+			"maxLength": 128
+		},
+		"v_tracker": {
+			"type": ["string", "null"],
+			"maxLength": 100
+		},
+		"v_collector": {
+			"type": "string",
+			"maxLength": 100
+		},
+		"v_etl": {
+			"type": "string",
+			"maxLength": 100
+		},
+		"user_id": {
+			"type": ["string", "null"],
+			"maxLength": 255
+		},
+		"user_ipaddress": {
+			"type": ["string", "null"],
+			"anyOf": [
+				{ "format": "ipv4" },
+				{ "maxLength": 128 }
```

Comment:
is there an ipv6 format?

Reply:
Hey @rzats, Ben is right, there's `ipv6` as well.